sklearn/linear_model/_cd_fast.pyx

```diff
@@ -506,7 +506,10 @@ def enet_coordinate_descent_gram(floating[::1] w,
     cdef unsigned int n_features = Q.shape[0]
 
     # initial value "Q w" which will be kept of up to date in the iterations
-    cdef floating[:] H = np.dot(Q, w)
+    # cdef floating[:] H = np.dot(Q, w)
+    cdef floating[:] H = np.zeros(n_features, dtype=dtype)
```

Comment:
Would suggest making this `empty` since it doesn't contribute to the computation below and is overwritten

```suggestion
    cdef floating[:] H = np.empty(n_features, dtype=dtype)
```